src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -671,52 +621,50 @@ def _set_dcp_params(self,
             self.fail("dst params require dst_type")
 
         # First, initialize a new dcp command
-        self.dcp_cmd = Dcp(self.hostlist_clients)
+        self.dcp_cmd = DcpCommand(self.hostlist_clients, self.workdir)
         self.dcp_cmd.get_params(self)
 
         # Set the source params
         if src_type == "POSIX":
-            self.dcp_cmd.set_dcp_params(
-                src=str(src_path))
+            self.dcp_cmd.set_params(
+                src_path=str(src_path))
         elif src_type == "DAOS_UUID":
-            param = self._format_daos_path(src_pool, src_cont, src_path)
-            self.dcp_cmd.set_dcp_params(
-                src=param)
+            self.dcp_cmd.set_params(
+                src_path=self.format_daos_path(src_pool, src_cont, src_path))
         elif src_type == "DAOS_UNS":
             if src_cont:
                 if src_path == "/":
-                    self.dcp_cmd.set_dcp_params(
-                        src=src_cont.path.value)
+                    self.dcp_cmd.set_params(
+                        src_path=src_cont.path.value)
                 else:
-                    self.dcp_cmd.set_dcp_params(
-                        prefix=src_cont.path.value,
-                        src=src_cont.path.value + src_path)
+                    self.dcp_cmd.set_params(
+                        daos_prefix=src_cont.path.value,
+                        src_path=src_cont.path.value + src_path)
 
         # Set the destination params
         if dst_type == "POSIX":
-            self.dcp_cmd.set_dcp_params(
-                dst=dst_path)
+            self.dcp_cmd.set_params(
+                dst_path=str(dst_path))
         elif dst_type == "DAOS_UUID":
-            param = self._format_daos_path(dst_pool, dst_cont, dst_path)
-            self.dcp_cmd.set_dcp_params(
-                dst=param)
+            self.dcp_cmd.set_params(
+                dst_path=self.format_daos_path(dst_pool, dst_cont, dst_path))
```

Comment:
(pylint-no-member) Instance of 'DataMoverTestBase' has no 'format_daos_path' member